content/en/blog/_posts/2024-12-11-Kubernetes-v1-32-Release/index.md

```diff
@@ -211,6 +245,42 @@ workloads.
 
 This work was done as part of [KEP #4818](https://github.com/kubernetes/enhancements/issues/4818) by [SIG
 Node](https://github.com/kubernetes/community/tree/master/sig-node).
+
+### DRA: Standardized network interface data for resource claim status
+
+This enhancement adds a new field that allows drivers to report specific device status data for each allocated object 
+in a resource claim. It also establishes a standardized way to represent networking devices information.
+
+This work was done as a part of 
+[KEP #4817](https://github.com/kubernetes/enhancements/issues/4817), by 
+[SIG Network](https://github.com/kubernetes/community/tree/master/sig-network).
+
+### New statusz and flagz endpoints for core components
+
+New `/statusz` and `/flagz` endpoints added for core components (consumable via API only) to enhance debuggability 
```

Comment:
OK to merge, but: what does “consumable via API only“ mean? Will I find these in OpenAPI and see SubjectAccessReviews when I query them (my guess is: no).

Try something like:
```markdown
You can enable two new  HTTP endpoints, `/statusz` and `/flagz`, for core components. These enhance cluster
debuggability
````

Reply:
Agreed!